tests/acceptance/features/webUIFiles/breadcrumb.feature

```diff
@@ -6,13 +6,11 @@ Feature: access breadcrumb
   Background:
     Given user "user1" has been created with default attributes
 
- @issue-2538
-  Scenario: Check breadCrumb for folder one level below the root folder when rootFolder is set
+  Scenario: Check breadCrumb for folder one level below the root folder when rootFolder is set with no value
     Given the property "rootFolder" has been set to "" in phoenix config file
     And user "user1" has logged in using the webUI
     When the user opens folder "simple-folder" using the webUI
-    #    Then breadcrumb for folder "simple-folder" should be displayed on the webUI
-    Then there should be no breadcrumb displayed on the webUI
+    Then breadcrumb for folder "simple-folder" should be displayed on the webUI
     When the user opens folder "simple-empty-folder" using the webUI
```

Comment:
@HariBhandari07 Should we still keep also the steps for "simple-empty-folder"? Or was it only temporarily until the issue is solved?

Reply:
@LukasHirt I think we can remove steps for simple-empty-folder as the scenario is only for folder one level below the root folder